src/main/java/dev/watchwolf/entities/ServerType.java

```diff
@@ -1,6 +1,6 @@
 package dev.watchwolf.entities;
 
 public enum ServerType {
```

Comment:
In [dev.watchwolf.tester.TestConfigFileLoader, line 62](https://github.com/rogermiranda1000/WatchWolf-Tester/blob/629f3155c00acad5c3d41c8cd80e5b14d688da8a/src/main/java/dev/watchwolf/tester/TestConfigFileLoader.java#L62) you read the server type specified in the config file and convert it into ServerType type. You'll have to change that `valueOf` to accept UpperCammelCase inputs.